docs/Node Operators/running-a-chainlink-node.md

```diff
@@ -16,10 +16,41 @@ It's important to note that nodes can fulfill requests for open APIs out-of-the-
 
 If you would like to provide data from an authenticated API, you can add an [external adapter](../external-adapters/) to enable connectivity through the Chainlink node.
 
-Hardware requirements are light. The only heavy part is you'll need a blockchain node connection. If you use a 3rd party (defined below), you can use a machine with as little as 10GB of storage and 4GB of RAM.
-
 ![Chainlink Node Diagram](/files/ab5762f-end-to-end-diagram.png)
 
+# Hardware Requirements
+
+## Chainlink Node
+
+Your Chainlink node should be run on a server that has a public IP address.
+
+### Minimum
+
+To get started running a Chainlink node, you will need a machine with at least **2 cores** and **4 GB of RAM**. 
+
+### Recommended 
+
+The requirements for running a Chainlink node scale as the number of jobs your node services also scales.  For nodes with over 100 jobs, you will need at least **4 cores** and **8GB of RAM**.  
```

Comment:
```
For nodes with over 100 jobs, you will need at least **4 cores** and **8GB of RAM**.  
```
Do we have a better metric for this like Job Runs per minute? I also assume the complexity of the job is a factor.

Reply:
This is the most detail we have available at the moment.  We can look into making things more specific in the future, but right now we don't have that info.